app/index.js

```diff
@@ -53,23 +53,26 @@ module.exports = yeoman.Base.extend({
           checked: true
         }, {
           name: 'Framework (Bootstrap 4)',
-          value: 'bootstrapSass',
+          value: 'bootstrap',
           checked: true
         }, {
           name: 'Tests (Mocha, Casperjs and Chai)',
           value: 'tests',
           checked: false
+        }, {
+          name: 'Contentful',
```

Comment:
You're right. I just hate having this question everytime for a feature we have set up once.

What if we just pass the option via a parameter? `$ yo toolbox --contentful`